Identity/Identity.swift

```diff
@@ -9,6 +9,7 @@
 import Foundation
 
 public protocol Identity {
+    var sso: SSOProvidable { get }
```

Comment:
Maybe something more explicit...
ssoProvider? ssoAccountDetailProvider?

Reply:
I would recommend leaving with the suffix `able`. From [Swift Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#protocols-describing-what-is-should-read-as-nouns)
> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).

Also it seems that this naming is used inside the SDK already, e.g. `ProductProvidable`, `PaymentProvidable`. 
Should it be `ssoAccountDetailProvider`...it depends if we want to mirror Android names, I think they have this pass: 
`identity.sso.SSOFeature`, but we don't use `Feature` anywhere, so to mirror it, I used just `sso`  Do you want me to change it?

Reply:
I meant more the variable name than the class name!
"sso" is a bit vague!

Reply:
Oh, got it. Agree. Renamed it